src/classes/error-message.ts

```diff
@@ -161,7 +164,7 @@ export class ErrorMessage {
    * @returns {this} - returns the instance for chaining
    */
   setUrl(url?: string) {
-    this.context.httpRequest.url = (is.string(url) ? url : '')!;
+    this.context.httpRequest.url = (typeof url === 'string' ? url : '')!;
```

Comment:
Can we extract this into method and call it throughout - just like we did when depending on Is but without depending on extra package?